frontend/src/app/lightning/node/node.component.html

```diff
@@ -75,13 +78,16 @@ <h1 class="mb-0 text-truncate">{{ node.alias }}</h1>
                 <div [ngStyle]="{'color': node.color}">{{ node.color }}</div>
               </td>
             </tr>
-            <tr *ngIf="node.country">
+            <tr>
               <td i18n="isp" class="text-truncate label">ISP</td>
-              <td>
+              <td *ngIf="node.as_number">
                 <a class="d-block text-wrap" [routerLink]="['/lightning/nodes/isp' | relativeUrl, node.as_number]">
                   {{ node.as_organization }} [ASN {{node.as_number}}]
                 </a>                
               </td>
+              <td *ngIf="!node.as_number">
```

Comment:
If there is no ASN, it could simply be missing from the geoip-db - we should differentiate between "unknown clearnet" and "hostname ends in .onion"